standard_rules/brute_force_by_ip.yml

```diff
@@ -5,6 +5,7 @@ DedupPeriodMinutes: 60
 DisplayName: Brute Force By IP
 Enabled: true
 LogTypes:
+  - Asana.Audit
```

Comment:
Might be nice to add a test case for this to ensure the data model is working as expected.

Reply:
Added :)